config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -0,0 +1,72 @@
+name: pangeo-hubs
+image_repo: "us-central1-docker.pkg.dev/two-eye-two-see/pilot-hubs-registry/base-user"
```

Comment:
This will probably not be pullable for this repo. Let's explicitly specify an image from github.com/pangeo-data/pangeo-docker-images/ under `singleuser` instead?